services/meta/association.md

```diff
@@ -81,10 +92,65 @@ _IMPORTANT_
   "type": "evt.association.report",
   "val_t": "object",
   "val": {
-    "group": "group_1",
```

Comment:
Are you changing type of object property? This is backwards incompatible change. You cannot do it unless it was an error in the documentation.

Reply:
"group" isn't used for Zigbee and this whole service hasn't been implemented in Zwave adapter at all. Is anything using this with the current group implementation?

Reply:
Ok.